### ISSUE TITLE
GFA output reverses node traversals

### ISSUE THREAD

**Issue by ekg** [vgteam/vg#1494]
Working on the tiny graph, I found a strange thing in the GFA output.

It seems that the path description shows each node traversal as its own reverse complement.

```
-> % vg construct \
    -v tiny/tiny.vcf.gz \
    -r tiny/tiny.fa >tiny.vg
```

The path 1-,3-,5- .... should be 1+,3+,5+...

```
-> % vg view tiny.vg
H       VN:Z:1.0
P       x       1-,3-,5-,6-,8-,9-,11-,12-,14-,15-       8M,1M,1M,3M,1M,19M,1M,4M,1M,11M
....
```

What might be the problem?

**Comment by edawson** [comment]
I'm on a roll this week.

I think this is a single line fix somewhere in `to_gfa`.

**Comment by edawson** [comment]
This was due to a missing `!`.

**Comment by edawson** [comment]
This is fixed in 1.8.0 - I'm going to close this. If somehow it pops back up please reopen.
